drivers/docker/driver_windows.go

```diff
@@ -15,3 +19,19 @@ func getPortBinding(ip string, port string) docker.PortBinding {
 func tweakCapabilities(basics, adds, drops []string) ([]string, error) {
 	return nil, nil
 }
+
+var containerAdminErrMsg = "running container as ContainerAdmin with is unsafe; change the container user, set task configuration to privileged or disable windows_allow_insecure_container_admin to disable this check"
+
+func (d *Driver) validateImageUser(user, taskUser string, driverConfig *TaskConfig) error {
+	// we're only interested in the case where isolation is set to "process"
+	// (it's also the default) and when windows_allow_insecure_container_admin
+	// is explicitly set to true in the config
+	if d.config.WindowsAllowInsecureContainerAdmin || driverConfig.Isolation == "hyper-v" {
+		return nil
+	}
+
+	if (user == "ContainerAdmin" || taskUser == "ContainerAdmin") && !driverConfig.Privileged {
```

Comment:
The `task.user` will override the user from the image manifest. Ex. if you have `task.user = "ContainerUser"` (safe) it'll override `"ContainerAdmin"` (unsafe) from the image and the container will run as "ContainerUser". It looks like we have it right in the docs, but I don't think we want this to be and "or" condition.

Reply:
I fixed in f32473e741d300031619ac6d4cd63d9dd8eaf68c, have a look if I got this correctly.